components/client/home/tag-line.js

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { Typography } from "@uoguelph/react-components/typography";
+import { Container } from "@uoguelph/react-components/container";
+import { Divider } from "@uoguelph/react-components/divider";
+
+export const TagLine = () => (
+  <Container className="flex flex-col py-4 items-center justify-center">
+    <Typography type="h1" as="h1" className="text-center!">
+      University of Guelph, Ontario, Canada
+    </Typography>
+
+    <span className="mt-9 mb-6 text-center text-2xl">Improve Life</span>
```

Comment:
DM Sans is missing from multiple spots
<img width="1681" height="421" alt="image" src="https://github.com/user-attachments/assets/5a095c78-1a20-4250-9ca7-4f74e6914a15" />